.vscode/cspell.json

```diff
@@ -50,6 +50,7 @@
     "EKUs",
     "Favicon",
     "gcovr",
+	"Giqu",
```

Comment:
What is this word? BTW, the spacing is misaligned.
If it is used just a few times in a very small scope ,can you use `// cspell: disable(-next)-line`?

Reply:
I see you only use it in unit tests? Can you use some known word instead, such as `username` or `johndoe`?

Reply:
i use giqu and goqu in tests , it's tradition. 
i applied the disable line approach

Reply:
> it's tradition

help us get in on this tradition, what's special about these? :)